packages/core/src/util/firmware.ts

```diff
@@ -2,13 +2,20 @@
 import MemoryMap from "nrf-intel-hex";
 import { ZWaveError, ZWaveErrorCodes } from "../error/ZWaveError";
 
-export type FirmwareFileFormat = "aeotec" | "otz" | "ota" | "hex";
+export type FirmwareFileFormat = "aeotec" | "otz" | "ota" | "hex" | "gecko";
```

Comment:
@AlCalzone What about making this an enum? So we prevent to have hardcoded strings

Reply:
I don't really see the benefit of this (breaking!) change when there is auto-completion and an exported type to help the discoverability.

Reply:
Ok no problem :)